server/parsec/config.py

```diff
@@ -187,6 +187,13 @@ class LogLevel(Enum):
     CRITICAL = logging.CRITICAL
 
 
+class AllowedClientAgent(Enum):
+    # Note the enum value is used in the PostgreSQL queries, so modifying it
+    # impacts the datamodel !
+    NATIVE_ONLY = "NATIVE_ONLY"
+    NATIVE_OR_WEB = "NATIVE_OR_WEB"
```

Comment:
Would add docstring/comments even if it's already described in other parts of the code.

```suggestion
class AllowedClientAgent(Enum):
    """
    Client agent allowed to connect to an organization.

    - `NATIVE_ONLY`: Only desktop client is allowed
    - `NATIVE_OR_WEB`: Desktop and web clients are allowed
    """
    # Note the enum value is used in the PostgreSQL queries, so modifying it
    # impacts the datamodel !
    NATIVE_ONLY = "NATIVE_ONLY"
    NATIVE_OR_WEB = "NATIVE_OR_WEB"
```

Reply:
:+1: done.